src/commands/rsync.ts

```diff
@@ -23,8 +23,13 @@ export const rsyncCommand = async () => {
     const wpPath = await vscode.window.showInputBox({
         prompt: 'Enter the remote path to upload the plugin contents to.',
         placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
+		ignoreFocusOut: true,
     });
 
+	if (!plugin || !wpPath) {
```

Comment:
Hmm, I think it'd be even better to break this up so if you cancel the first prompt it doesn't even bother asking you for the path.

Reply:
Agreed, updated